### ISSUE TITLE
[Bug] Restricted Musixmatch lyrics pop up unreadable 

### ISSUE THREAD

**Issue by Lbdawg** [whoeevee/EeveeSpotify#265]
### Is this issue appropriate?

- [X] I have read the document in its entirety

### Is this issue unique to EeveeSpotify?

- [X] I was unable to reproduce my issue using the stock Spotify app

### Have you searched the existing issues?

- [X] This is a unique issue

### Describe the bug.

It's not a major problem, but I thought it was worth mentioning so you can consider fixing it in a future update.

https://github.com/user-attachments/assets/a1db52f6-6fb3-486a-bb60-9aa2a723d799

**Comment by asdfzxcvbn** [comment]
is this the latest version

**Comment by Lbdawg** [comment]
Yes it’s the latest version

**Comment by Lbdawg** [comment]
> is this the latest version

Yes it’s the latest version

**Comment by asdfzxcvbn** [comment]
yeah could reproduce this, leaving this issue open until it's fixed

**Comment by asdfzxcvbn** [comment]
@hugeBlack you have a jailbroken ipad right? could you take a shot at fixing this?

**Comment by hugeBlack** [comment]
I'll look into this

**Comment by hugeBlack** [comment]
Can't confirm this issue. I built EeveeSpotify with Spotify 8.9.58 and the Unauthorized Musixmatch popup displayed correctly. I tried this in both light and dark mode and both are fine.
I'm on iPad OS 17.5.1 and iPad Pro 2017

![IMG_0500](https://github.com/user-attachments/assets/7d540ecc-4ddc-4fef-9ad5-b9f75c1962b5)
![IMG_0501](https://github.com/user-attachments/assets/ecbbcf5f-7fbd-4aec-b5ac-8e147408c52e)
